.github/workflows/ci.yml

```
@@ -324,14 +324,41 @@ jobs:
     name: "Deploy SAW documentation to GitHub Pages"
     needs: [build-docs]
     runs-on: ${{ matrix.os }}
-    # Only run for pushes to `master` and version tags, but not in forks.
-    # This is to avoid issues with missing permissions on GITHUB_TOKEN, needed
-    # when actually deploying the documentation to the `gh-pages` branch.
     #
-    # Note that pushing a tag results in a github.ref of the form
-    # `refs/tags/...`: Even if the tag refers to `HEAD` of a `release-*` branch,
-    # only a workflow for the `refs/tags/...` ref will be started.
-    if: github.event_name == 'push' && needs.config.outputs.release == 'false' && github.repository_owner == 'GaloisInc'
+    # We are only run at all on branches matching `master`, branches
```

I don't think this sentence came out as intended. Perhaps something like:

```suggestion
    # We only run this job on branches matching `master`, branches
```

---

Fixed, I think.